### PR TITLE
Rate limit modifications

### PR DESCRIPTION
**Pull request by ux-AylonS**
New Rate Limit modifications

### CODE REVIEW

File: src/doc-editors/__tests__/RateLimitsEditor.spec.ts [hunk 1]
```diff
@@ -147,9 +147,9 @@ describe('RateLimitsEditor.vue', () => {
     })
 
     test('should have event limit option component with correct data', () => {
-      const wantedType = Object.keys(rateLimitsDocs[0].pairwith)[0]
-      const wantedValue = Object.values(rateLimitsDocs[0].pairwith)[0]
-      const limitOptionComponent = wrapper.findAllComponents(LimitOption).at(1)
+      const wantedType = Object.keys(rateLimitsDocs[0].key[0])[0]
```
Comment (Aviv-Galmidi) [inline src/doc-editors/__tests__/RateLimitsEditor.spec.ts:150]: That test is for `pairtwith` not `key` (Event in the UI)

File: src/doc-editors/__tests__/RateLimitsEditor.spec.ts [hunk 2]
```diff
@@ -230,13 +230,13 @@ describe('RateLimitsEditor.vue', () => {
...
       limitOptionsComponent.vm.$emit('remove', 1)
-      expect(wrapper.vm.localDoc.key.length).toEqual(1)
+      expect(wrapper.vm.localDoc.key.length).toEqual(3)
     })
 
     test('should not be able to remove key when only one key exists', async () => {
       const limitOptionsComponent = wrapper.findComponent(LimitOption)
       limitOptionsComponent.vm.$emit('remove', 1)
-      expect(wrapper.vm.localDoc.key.length).toEqual(1)
+      expect(wrapper.vm.localDoc.key.length).toEqual(2)
```
Comment (Aviv-Galmidi) [inline src/doc-editors/__tests__/RateLimitsEditor.spec.ts:239]: Read the test name again. We want to test that we DO NOT remove a key when only one key exists. If we start with 3 keys by default you can remove 2 of them before testing the last one can't be removed, or set the initial data for this test to be a rate limit with only one key

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/doc-editors/SecurityPoliciesEditor.vue [hunk 3]
```diff
@@ -70,7 +70,7 @@
                     :title="mapEntry.name">
                   {{ mapEntry.name }}
                 </td>
-                <td class="is-size-7 width-360px ellipsis entry-match"
+                <td class="is-size-7 width-200px ellipsis entry-match"
```
[the remark this answers is not in the document]
Comment (ux-AylonS) [reply src/doc-editors/SecurityPoliciesEditor.vue:73]: done. the problem is with small screens under 1300px

File: src/assets/DatasetsUtils.ts [hunk 4]
```diff
@@ -180,6 +180,7 @@ const newDocEntryFactory: { [key: string]: Function } = {
       'match': `${id}.example.com`,
       'map': [
         {
+          'id': id,
```
Comment (ux-AylonS) [inline src/assets/DatasetsUtils.ts:183]: Is this should be here?

File: public/static/bulma.css [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
...tarea::-moz-placeholder{color:rgba(54,54,54,.3)}.input::-webkit-input-placeholder,.select select::-webkit-input-placeholder,.textarea::-webkit-input-placeholder{color:rgba(54,54,54,.3)}.input:-moz-placeholder,.select select:-moz-placeholder,.textarea:-moz-placeholder{color:rgba(54,54,54,.3)}.input:-ms-input-placeholder,.select select:-ms-input-placeholder,.textarea:-ms-input-placeholder{color:rgba(54,54,54,.3)}.input:hover,.is-hovered.input,.is-hovered.textarea,.select select.is-hovered,.select select:hover,.textarea:hover{border-color:#b5b5b5}.input:active,.input:focus,.is-active.input,.is-active.textarea,.is-focused.input,.is-focused.textarea,.select select.is-active,.select select.is-focused,.select select:active,.select select:focus,.textarea:active,.textarea:focus{border-color:#3273dc;box-shadow:0 0 0 .125em rgba(50,115,220,.25)}.input[disabled],.select fieldset[disabled] select,.select select[disabled],.textarea[disabled],fieldset[disabled] .input,fieldset[disabled] .select select,fieldset[disabled] .textarea{background-color:#f5f5f5;border-color:#f5f5f5;box-shadow:none;color:#7a7a7a}.input[disabled]::-moz-placeholder,.select fieldset[disabled] select::-moz-placeholder,.selec
```
Comment (Aviv-Galmidi) [inline public/static/bulma.css:1]: What did you change in Bulma?

File: src/assets/DatasetsUtils.ts (hunk 4 above)
Comment (Aviv-Galmidi) [reply src/assets/DatasetsUtils.ts:183]: Yes that was part of the task, adding `id` to map entry in Security Policy

**Review by Aviv-Galmidi — COMMENTED** [review]
Need to resolve conflicts

File: src/assets/DatasetsUtils.ts (hunk 4 above)
Comment (Aviv-Galmidi) [reply src/assets/DatasetsUtils.ts:183]: Yes that was part of the task, adding id to map entry in Security Policy

File: src/components/securityPoliciesConnections.vue [hunk 6]
```diff
@@ -1,155 +1,156 @@
...
-                    @keypress.enter="setEntryDeleteIndex(-1)">
-                    <i class="fas fa-times"></i> Cancel
-                </a>
-            </span>
-        </td>
-      </tr>
-      </tbody>
-    </table>
-  </div>
+<div>
```
Comment (Aviv-Galmidi) [inline src/components/securityPoliciesConnections.vue:2]: Fix indents in this page

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**